packages/cli/src/events/telemetry-event-relay.ts

```diff
@@ -753,6 +759,52 @@ export class TelemetryEventRelay extends EventRelay {
 		});
 	}
 
+	private sessionStarted({ pushRef }: RelayEventMap['session-started']) {
+		this.telemetry.track('Session started', { session_id: pushRef });
+	}
+
+	private async instanceStopped() {
+		const timeoutPromise = new Promise<void>((resolve) => {
+			setTimeout(resolve, 3000);
+		});
+
+		return await Promise.race([timeoutPromise, this.telemetry.trackN8nStop()]);
```

Comment:
I don't think it makes sense to have the race here anymore, as nobody is awaiting this. We could simplify and only class `this.telemetry.trackN8nStop()`